src/components/UserField/UserField.js

```diff
@@ -1,24 +1,46 @@
 import React from 'react';
-import './userField.css'
+import './userField.css';
 
 export default class UserField extends React.Component {
     constructor() {
         super();
     }
-    
     render() {
         const { users } = this.props;
         return (
             <ul>
                 {users.map((item, index) => {
                     return (
-                        <li key={index}>
-                            {item.name} - {item.surname} - {item.phone} - {item.email} 
-                            {/* <img src = {this.props.url} alt='something'/> */}
+                        <li className="list" key={index}>
+                            <div>
+                                <img src={item.photo} alt="image" style={{height: "50px"}}/>
+                            </div>
+                            <div>
+                                {item.isOnline ? <i className="fas fa-circle green"></i> : <i className="fas fa-circle gray"></i>}
```

Comment:
if your tag or component have no children write like this` <i className="fas fa-circle green" />